pylib/gyp/__init__.py

```diff
@@ -463,8 +463,18 @@ def gyp_main(args):
         metavar="TARGET",
         help="include only TARGET and its deep dependencies",
     )
+    parser.add_argument(
+        "-V",
+        "--version",
+        dest="version",
+        action="store_true",
+        help="Show the version and exit.",
+    )
 
     options, build_files_arg = parser.parse_args(args)
+    if options.version:
+        print("v0.13.0")
```

Comment:
Can this be done reading https://github.com/nodejs/gyp-next/blob/main/setup.py ?
Currently auto release may not update this timely.

Reply:
The version number should not be stored in an executable file like `setup.py` but in a config file like `pyproject.toml` or `setup.cfg`.  I will make that change...